src/scenes/GameScene.js

```diff
@@ -47,13 +49,24 @@ export default class Game extends Phaser.Scene {
     );
 
     this.groundLayer = this.tilemap.createLayer("Ground", tilesetOutside);
-    this.tilemap.createLayer("Buildings", tilesetOutside);
+    this.tilemap.createLayer("OnGround", tilesetOutside);
+    this.tilemap.createLayer("Collides", tilesetOutside);
     this.tilemap.createLayer("Above", tilesetOutside);
+  }
 
-    this.socket = io();
+  extractMapFromPhaserObject() {
+    // find better way to do that
```

Comment:
Please create an issue and add it here like this:
```suggestion
    // TODO (#xx) find better way to do that
```